### PR TITLE
Improve changes uri creation

### PR DESCRIPTION
**Pull request by hrsh7th**

### CODE REVIEW

File: autoload/jetpack.vim [hunk 1]
```diff
@@ -288,20 +285,12 @@ function! s:display() abort
...
-    if pkg.progress.type ==# s:progress_type.update
-      let from_to = matchstr(output, 'Updating\s*\zs[^\n]\+')
-      if from_to !=# ''
-        call s:setbufline(line_count, printf('  Changes %s/compare/%s', pkg.url, from_to))
-        let line_count += 1
-      endif
-    endif
-
+    call s:setbufline(line_count, printf('%s %s', msg[pkg.progress.type], pkg.name))
+    let line_count += 1
```
Comment (tani) [inline autoload/jetpack.vim:293]: I would like to hear the summary of this simplification.

**Comment by tani** [comment]
Thank you for the beautiful contribution.
The most of changes look smarter than before.

File: autoload/jetpack.vim (hunk 1 above)
Comment (hrsh7th) [reply autoload/jetpack.vim:293]: The previous implementation has a problem that might occur if the repo will rebased. (The `Updating...` message is omitted in such case in my environment.)


It seems the `From %URL% %FROM%...%TO%` is always displayed so I changed to use this message here. (If we hope to support it perfectly, we should use `git log` command for displaying the change history.)

Comment (tani) [reply autoload/jetpack.vim:293]: It's reasonable. Thank you for the explanation.

**Pull request merged**